style/style.css

```diff
@@ -0,0 +1,319 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+  box-sizing: border-box;
+}
+body {
+  background-color: #ececec;
+}
+header {
+  background-image: url(../images/header-image.jpg);
+    background-size:100% 100%;
+}
+.adoptadog-logo img{
+
+  display:inline;
+  position: relative;
+  left: 2rem;
+  width:50%;
+  padding: 7.5rem;
+  top: 0.625rem;
+}
+.dog-adoption-agency {
+ color:#81b741;
+ font-size:1.875rem;
+
+ display: inline;
+ position: relative;
+ left: 11rem;
+ bottom:8rem;
+
+
+}
+.price {
+
+ display: inline;
+ position:absolute;
+ right:8rem;
+ top: 7rem;
+ color:white;
+}
+.checkout {
+
+display:inline;
+position:absolute;
+right:8.5rem;
+top:8.8rem;
+background-color:#81b741;
+width:7%;
+height: 1.5rem;
+text-align: center;
+
+
+}
+.checkout a {
+  color:white;
+  text-decoration: none;
+}
+
+
+
+
+
+
+.top-nav {
+    background-color:  #d4d4d4;
+
+}
+ ul {
+
+  overflow: hidden;
+  list-style-type: none;
+  margin-left: 2.5rem;
+  padding:1.8rem;
+ }
+  li {
+  float: left;
+
+ }
+li  a {
+  display: block;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  color: #005580;
+  font-weight:bold;
+}
+.box {
+  display: flex;
+  flex-direction:row;
+  }
+.box img {
+  margin-top:2rem;
+  margin-left:5rem;
+}
+.content {
+  margin-right:2rem;
+  margin-top:-2rem;
+  margin-left:2rem;
+  padding: 2.875rem;
+  line-height: 2.3rem;
+}
+.content h1 {
+  margin-bottom:1.2rem;
+  margin-top: 1rem;
+  color: #094160;
+  font-weight: bold;
+  font-size: 3rem;
+}
+.content p {
+  font-size:1.7rem;
+  word-spacing:1rem;
+  letter-spacing: 0.02rem;
+  color:	 #666666;
+}
+.row {
+  margin: 2.7rem;
+}
+.row:after {
+ content: "";
+ display: table;
+ clear: both;
+}
+.column {
+ float: left;
+ width: 25%;
+}
+
+.card-content {
+  border: 0.125rem solid gray;
+  width: 85%;
+  box-shadow:0 0.5rem 0.625rem 0 #888888;
+  margin: 2rem;
+}
+
+.container {
```

Comment:
Padding is a little large at different resolutions, Try experimenting with smaller values.